templates/accounts/resend_activation_mail.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Resend activation mail {% endblock %}
+
+
+{% block content %}
+
+<div class="container pt-3" style="margin-top: 100px;">
+    <div class="text-center mt-4 p-0 p-sm-3" style="margin: auto; max-width: 500px;">
+        <h1 class="mt-2 font-weight-bolder" style="margin-bottom: 30px;">Resend Activation mail</h1>
+        <div class="form-outline">
+            <form method="post">
```

Comment:
The email field in form looks too long. So, add `style="margin: auto; max-width: 400px;` in this element.

Also, you should always write HTTP Requests in all-caps (`POST`), as a convention.